tests/integration_tests/utils.py

```diff
@@ -922,11 +925,55 @@ def filter(stats):
                         assert name1 == name2
                         assert np.isclose(
                             metric1, metric2, rtol=1e-03, atol=1e-04
-                        ), f"metric {name1}: {metric1} != {metric2}"
+                        ), f"metric {name1} for feature {k1}: {metric1} != {metric2}"
 
         return model
 
 
+def all_required_metrics_exist(
```

Comment:
nit: Rename to `assert_all_required_metrics_exist` or `check_all_required_metrics_exist` to make it more obvious that an assert is called.